public-api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -100,6 +108,109 @@ do
     bq --project_id=$OUTPUT_PROJECT load --quote='"' --source_format=CSV --skip_leading_rows=1 --max_bad_records=10 $OUTPUT_DATASET.$t $csv_path/$t.csv
 done
 
+if [ "$SEARCH_VAT" = true ]; then
+  bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+  "DROP MATERIALIZED VIEW IF EXISTS \`$OUTPUT_PROJECT.$GENOMICS_DATASET.wgs_variant_vid_cluster_mv\`"
+
+  bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+  "DROP TABLE IF EXISTS \`$OUTPUT_PROJECT.$GENOMICS_DATASET.wgs_variant\`"
+
+  bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+  "CREATE TABLE \`$OUTPUT_PROJECT.$GENOMICS_DATASET.wgs_variant\` as
```

Comment:
This should be a clustered table correct? In order for the materialized view clustering to work, the base table has to be clustered as well.